src/AddImageChecks.cpp

```diff
@@ -201,10 +201,10 @@ Stmt add_image_checks(Stmt s,
                 Expr query_buf = Variable::make(type_of<struct buffer_t *>(),
                                                 param.name() + ".bounds_query." + extern_user);
                 for (int j = 0; j < dimensions; j++) {
-                    Expr min = Call::make(Int(32), Call::extract_buffer_min,
-                                          {query_buf, j}, Call::Intrinsic);
-                    Expr max = Call::make(Int(32), Call::extract_buffer_max,
-                                          {query_buf, j}, Call::Intrinsic);
+                    Expr min = Call::make(Int(32), "_halide_buffer_get_min",
```

Comment:
Fixed. I added a section in Call for named string constants for runtime functions. We can add/move things there going forward.